src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -732,7 +732,11 @@ namespace winrt::TerminalApp::implementation
 
     void TerminalPage::_SendFeedbackOnClick(const IInspectable& /*sender*/, const Windows::UI::Xaml::Controls::ContentDialogButtonClickEventArgs& /*eventArgs*/)
     {
-        ShellExecute(nullptr, nullptr, L"https://github.com/microsoft/terminal/issues", nullptr, nullptr, SW_SHOW);
+#if defined(WT_BRANDING_RELEASE)
+        ShellExecute(nullptr, nullptr, L"https://aka.ms/terminal-feedback-hub", nullptr, nullptr, SW_SHOW);
```

Comment:
We should be able to use `https://go.microsoft.com/fwlink/?linkid=2125419` for this! It's the link we used to use, afterall... and we should endeavor to always use fwlinks in our final product because they can do language-based redirection and stuff.

Reply:
```suggestion
        ShellExecute(nullptr, nullptr, L"https://go.microsoft.com/fwlink/?linkid=2125419", nullptr, nullptr, SW_SHOW);
```